### DISCUSSION TITLE
Does the pre-built tensorflow_cc supports AVX2/AVX512 extensions?

### DISCUSSION THREAD

**Discussion by parveenduhan** [ika-rwth-aachen/libtensorflow_cc#12] [General]
Also, does the [tensorflow-2.13.0-cp38-cp38-linux_x86_64.whl](https://github.com/ika-rwth-aachen/libtensorflow_cc/releases/download/v2.13.0/tensorflow-2.13.0-cp38-cp38-linux_x86_64.whl) built with additional support for AVX extensions?

**Comment by lreiher** [comment]
The [compile flags](https://github.com/ika-rwth-aachen/libtensorflow_cc/blob/main/docker/Dockerfile#L57) would suggest a yes, however, I don't have access to an Intel CPU to check.
